### PR TITLE
Add gRPC-Web Browser example

### PR DESCRIPTION
**Pull request by JamesNK**
Builds on top of the [gRPC-Web PR](https://github.com/grpc/grpc-dotnet/pull/695).

Actual example code is [here](https://github.com/grpc/grpc-dotnet/pull/703/commits/2dddf2d0f67ae593ac06bab0ea72f47106efc90d).

### CODE REVIEW

**Comment by JamesNK** [comment]
Deployed: https://grpcweb.azurewebsites.net/

**Comment by jtattermusch** [comment]
@stanley-cheung  can you please review the JavaScript grpc-web client portion of the code here: https://github.com/grpc/grpc-dotnet/pull/703/commits/2dddf2d0f67ae593ac06bab0ea72f47106efc90d

**Comment by stanley-cheung** [comment]
LGTM for the grpc-web part. Just a note that the .js files are generated code.

**Comment by jtattermusch** [comment]
I just LGTMd https://github.com/grpc/grpc-dotnet/pull/695, let's merge that one and rebase this PR for easier review?

**Comment by JamesNK** [comment]
Rebased

File: examples/README.md [hunk 1]
```diff
@@ -146,3 +146,14 @@ The vigor example shows how to integrate [ASP.NET Core health checks](https://do
 * Hosting gRPC Health Checking Protocol service
 * Integrate [ASP.NET Core health checks](https://docs.microsoft.com/aspnet/core/host-and-deploy/health-checks) with gRPC health checks
 * Calling service with `Grpc.HealthCheck` client
+
+## [Browser](./Browser)
+
+The browser example shows how to use [gRPC-Web](https://github.com/grpc/grpc-web) with ASP.NET Core to call a gRPC service from a browser. This example requires [npm and NodeJS](https://nodejs.org/) to be installed on your computer.
```
Comment (jtattermusch) [inline examples/README.md]: just curious: are you going to add an example for .NET grpc-web client using Blazor/WebAssembly as well? It's an interesting piece of technology in that it allows one to use the same gRPC C# client API regardless whether you're running in the browser or if you're a microservice in a datacenter.

Comment (JamesNK) [reply examples/README.md]: I will add it as a separate sample.

**Pull request merged**